.github/labeler.yml

```diff
@@ -191,10 +224,11 @@ utils.masked:
     - docs/utils/masked/**/*
 
 visualization:
-- any:
+- all:
   - changed-files:
     - any-glob-to-any-file:
       - '**/visualization/**/*'
+    - all-globs-to-any-file:
       - '!**/visualization/wcsaxes/**/*'
```

Comment:
@neutrinoceros , maybe you can tell me if this syntax is correct. Conceptually, I want the rule to say "find all the `**/visualization/**/*` except if it is `**/visualization/wcsaxes/**/*`" because wcsaxes has its own separate label.

Reply:
I don't think this is doing what you want (though it is syntactically valid dans most likely schema-valid); if I understand correctly the documentation (that's a big if) having a single glob in either `- all-globs-to-any-file` or `- any-glob-to-any-file` is exactly equivalent, and both groups are independent, not complementary.
I already experimented with that earlier and this is my verified solution:
https://github.com/neutrinoceros/astropy/pull/12

I don't know if changing the top level `- any` to `- all` makes any difference here. All I can say is that I didn't need it.

Reply:
Thanks. Lemme double check. Maybe I need `all-globs-to-all-files` as documented in their readme.